### PR TITLE
Add -t flag to target the space containing the route

### PR DESCRIPTION
**Pull request by cnelson**
I've gotten tired of copying and pasting this output into the `cf target` command. 😁

### CODE REVIEW

**Review by afeld — CHANGES_REQUESTED** [review]
I love it! Mind adding a note to the README? Also, sorry for the slow response!

File: README.md [hunk 1]
```diff
@@ -14,9 +14,15 @@ This is a Cloud Foundry CLI plugin to find the application a given hostname/doma
...
-$ cf lookup-route <my.domain.com>
+$ cf lookup-route <my.example.com>
 Bound to:
 <org>/<space>/<app>
-$ cf lookup-route <unknown.domain.com>
+
+# use -t to target the org/space containing the route
+$ cf lookup-route -t <my.example.com>
+Bound to:
+<org>/<space>/<app>
```
Comment (afeld) [inline README.md:24]: Sorry, one more thing: mind adding a message of "Changed target to <org>/<space>"?

File: main.go [hunk 2]
```diff
@@ -55,6 +57,14 @@ func (c *BasicPlugin) Run(cliConnection plugin.CliConnection, args []string) {
 		}
 		fmt.Println(org.Entity.Name + "/" + space.Entity.Name + "/" + app.Entity.Name)
 	}
+
+	if *target {
+		_, err = apps[0].Target(cliConnection)
```
Comment (afeld) [inline main.go:63]: Should we do anything special if the route is bound to multiple apps?

Comment (cnelson) [reply main.go:63]: Is it possible to bind a route to multiple apps in different spaces?  When I tried I received: `Server error, status code: 400, error code: 210003, message: The host is taken: example`  but perhaps there's another way of doing it that I'm unaware.

Comment (afeld) [reply main.go:63]: Ah, good point! Works for me.

**Review by afeld — APPROVED** [review]

**Pull request merged**